src/shogun/mathematics/linalg/LinalgBackendBase.h

```diff
@@ -228,7 +228,20 @@ class LinalgBackendBase
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_SET_CONST, SGVector)
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_SET_CONST, SGMatrix)
 	#undef BACKEND_GENERIC_SET_CONST
-	
+
+	/**
+	 * Wrapper method that sets const values to vectors or matrices.
```

Comment:
I am a nagger, i know.
BUT we should not offer such a method. It is extremely cache unfriendly -- matrices are stored in column major format in Shogun. This is why if you change a single element of all rows, this totally screws up the prefetchers hard work.
Devs should be forced to set the columns to constant values, and then potentially transpose afterwards.
@vigsterkr @lambday thoughts?